crates/bevy_pbr/src/render_graph/pbr_pipeline/pbr.frag

```diff
@@ -342,7 +342,7 @@ void main() {
 
     // accumulate color
     vec3 light_accum = vec3(0.0);
-    for (int i = 0; i < int(NumLights.x) && i < MAX_LIGHTS; ++i) {
+    for (int i = 0; i < int(NumLights.x); ++i) { // No need to check against MAX_LIGHTS as NumLights will not exceed it
```

Comment:
Put this comment on a separate line? This line is kinda long.

Reply:
Done.